flamedisx/x1t_sr1.py

```diff
@@ -53,37 +162,61 @@ def add_extra_columns(self, d):
                           d['z'].values]))
 
     @staticmethod
-    def electron_detection_eff(drift_time, *, elife=650e3, extraction_eff=0.96):
+    def electron_detection_eff(drift_time, *, elife=def_elife, extraction_eff=def_extract_eff):
         #TODO: include function for elife time dependency
         return extraction_eff * tf.exp(-drift_time / elife)
 
     @staticmethod
-    def electron_gain_mean(s2_relative_ly, *, g2=11.4/(1.-0.63)/0.96):
+    def electron_gain_mean(s2_relative_ly, *, g2=def_g2/(1.-def_frac_top)/def_extract_eff):
         return g2 * s2_relative_ly
 
     @staticmethod
-    def electron_gain_std(s2_relative_ly, *, g2=11.4/(1.-0.63)/0.96):
-        return g2*0.96*0.25+0.*s2_relative_ly
+    def electron_gain_std(s2_relative_ly, *, g2=def_g2/(1.-def_frac_top)/def_extract_eff):
+        return g2*def_extract_eff*0.25+0.*s2_relative_ly
 
     #TODO: implement better the double_pe_fraction or photon_detection_efficiency as parameter
     @staticmethod
-    def photon_detection_eff(s1_relative_ly, g1 =0.142):
-        #g1 = 0.142 from paper
-        mean_eff= g1 / (1. + 0.219)
+    def photon_detection_eff(s1_relative_ly, g1 =def_g1):
+        mean_eff= g1 / (1. + def_p_dpe)
         return mean_eff * s1_relative_ly
 
+    def s1_acceptance(self, s1, photon_detection_eff, photon_gain_mean, mean_eff=def_g1 / (1 + def_p_dpe), 
+            cs1_min=3, cs1_max=70):
```

Comment:
Could consider defining these as DEFAULT_CS1_MIN along with all the other defaults. Or leave all defaults defined with the function itself (which is not so bad) -- but mixing the two can be confusing

Reply:
Parameters that are used across more than 1 function(with the exception of elife and drift velocity, since they are important and would be good to have them visible at one glance) are defined as global variables at the top. Decided to keep cs1_min, cs1_max, cs2_min, cs2_max defined within the function for now.